loopchain/blockchain/blocks/v0_3/block_builder.py

```diff
@@ -176,6 +188,28 @@ def _build_hash(self):
         block_prover = BlockProver(leaves, BlockProverType.Block)
         return block_prover.get_proof_root()
 
+    def build_peer_id(self):
```

Comment:
BlockBuilder will have `signer`. Then `build_peer_id` method does not needed. `signer` has address(id)